Array in Javascript/README.md

```diff
@@ -108,6 +156,41 @@ No other element has been shifted, so it’s an O(1) runtime.
 Full code can be found at [here](src/deletion.js)
 ### Micro-Challenge
 How to remove multiple elements from array in JavaScript ? Give it a try create a PR and add it here as Micro-Challenge Solution
```

Comment:
Remove the following:
`create a PR and add it here as Micro-Challenge Solution`